public/test/helpers/observableTester.ts

```diff
@@ -0,0 +1,68 @@
+import { Observable } from 'rxjs';
+
+interface ObservableTester<T> {
+  observable: Observable<T>;
+  done: jest.DoneCallback;
+}
+
+interface SubscribeAndExpectOnNext<T> extends ObservableTester<T> {
+  expect: (value: T) => void;
+}
+
+interface SubscribeAndExpectOnComplete<T> extends ObservableTester<T> {
+  expect: () => void;
+}
+
+interface SubscribeAndExpectOnError<T> extends ObservableTester<T> {
+  expect: (err: any) => void;
+}
+
+export const observableTester = () => {
```

Comment:
Why not making `observableTester` an object instead of a function? Are you expecting it to be configurable at some point?

Reply:
No specific reason, just a pattern I use, I suppose. Makes it immutable, too I guess.